src/index.js

```diff
@@ -0,0 +1,28 @@
+import './style.css';
+import refresh from './assets/refresh.png';
+
+import { getToDos, addToDo, createToDo } from './module/script.js';
+import clearToDoItems from './module/interactive.js';
+
+const titleDiv = document.querySelector('.title_class');
+const clearCompleted = document.querySelector('.clear');
+const input = document.querySelector('.desc');
+
+const refreshB = document.createElement('img');
+refreshB.classList.add('reload');
```

Comment:
Kindly Use simple explanatory variable names, using simple names will help even others to understand your code easily and avoid the WTF